app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    conditions = command.split()
```

Comment:
Check command length, it should equal exactly to 3 parts, and the command name should be exactly "mv"